### DISCUSSION TITLE
GitHub Universe session

### DISCUSSION THREAD

**Discussion by AsnaKhan1** [community/community#74561] [GitHub Education] [answered]
### Hi everyone,

could someone please guide me as to how to attend the live session online at github universe?

**Comment by gitstua** [comment, marked as answer]
👋🏽  Hi, thanks for the question to our community @AsnaKhan1! 

To view the live sessions at GitHub Universe you can register for free at [githubuniverse.com](https://githubuniverse.com/)

**Comment by AsnaKhan1** [comment]
I have already registered, but when I am going to my calendar where I can
see my scheduled sessions I am not getting the option to join that live
session. I need help regarding that part.

**Reply by RohithSml** [reply]
+1 
I have also checked the main landing page after I registered earlier on, and it states "Livestream has not started."
